pytest-embedded/pytest_embedded/plugin.py

```diff
@@ -349,9 +350,15 @@ def pytest_addoption(parser):
         '- idf: auto-detect more app info with idf specific rules, auto flash-in\n'
         '- jtag: openocd and gdb\n'
         '- qemu: use qemu simulator instead of the real target\n'
+        '- arduino: auto-detect more app info with arduino specific rules, auto flash-in\n'
         'All the related CLI options are under the groups named by "embedded-<service>"',
     )
     base_group.addoption('--app-path', help='App path')
+    base_group.addoption('--build-dir', help='build directory under the app_path. (Default: "build")')
```

Comment:
`build_dir` move to base `App` is a good idea. However I think we need to add this argument to all `App` and its sub-classes. Now the base `App` doesn't have this attribute.